edx_django_utils/cache/tests/test_middleware.py

```diff
@@ -15,7 +15,6 @@
 
 class TestRequestCacheMiddleware(TestCase):  # pylint: disable=missing-class-docstring
 
-    @override_settings(MIDDLEWARE=['edx_django_utils.cache.middleware.RequestCacheMiddleware'])
```

Comment:
**Reviewer Note:** FYI: All of these `override_settings of `MIDDLEWARE` deletions are related to removing `_check_middleware_dependencies`.
Please mark as resolved once you've read this.